viewer_clients/cmdline_client/cmd_line_client.py

```diff
@@ -466,51 +486,55 @@ def register_client_command_line(argument_parser):
     subparsers = argument_parser.add_subparsers()
 
     # List runs.
-    listruns_parser = subparsers.add_parser('runs', help='Get the run data.')
+    listruns_parser = subparsers.add_parser('runs',
+                                            formatter_class=ADHFormatter,
+                                            help='Get the run data.')
     listruns_parser.add_argument('--host', type=str, dest="host",
                                  default='localhost',
                                  help='Server host.')
     listruns_parser.add_argument('-p', '--port', type=str, dest="port",
-                                 default=11444,
-                                 required=True, help='HTTP Server port.')
+                                 default="8001", help='HTTP Server port.')
```

Comment:
I made this a string because the type of it is `type=str`. This was a bug before because if the user did not provide a port as an argument the type of default value would be integer, which would cause errors in other modules. Most of the code depends on that the type of the port is string.
Further refactoring is needed in multiple modules to handle the integer type properly.